UnityProject/Assets/Scripts/Camera/FieldOfViewTiled.cs

```diff
@@ -163,24 +164,29 @@ IEnumerator FovProcessing()
     //Runs on Worker Thread:
     public void UpdateSightSourceFov()
     {
+        if (nextShrouds != nearbyShroudsInWorkerThread)
+        {
+            nearbyShroudsInWorkerThread = new ReadOnlyCollection<Vector2>(nextShrouds);
```

Comment:
No need to clone this anymore, the collection is readonly so it can't be modified by other threads.

Reply:
`nearbyShroudsInWorkerThread = nextShrouds;`